### PR TITLE
feat(TDI-44040) : HTML type added

### PR DESCRIPTION
**Pull request by clesaec**
HTML type added in read file.

### CODE REVIEW

File: common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelReader.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+            if (headers >= 1) {
+                this.parseHeaderRow(rowIterator, headers);
+            }
+
+            return IteratorComposer.of(rowIterator).skipFooter(configuration.calcFooter())
+                    .map((Row row) -> this.toRecord.toRecord(row)).closeable(currentWorkBook).build();
+        } catch (IOException exIO) {
+            log.error("Error while reading excel input", exIO);
+            throw new UncheckedIOException("Error while reading excel input", exIO);
```
Comment (undx) [inline common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelReader.java:57]: i18n as was discussed yesterday ;-)

File: common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelRecordReader.java [hunk 2]
```diff
@@ -40,65 +31,25 @@
...
-                rows.next();
-            }
-            final Row headerRow = rows.next();
-            return this.toRecord.inferSchema(headerRow, true);
-        }
-        return null;
-    }
-
     @Override
     public void close() {
```
Comment (undx) [inline common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelRecordReader.java:53]: remove empty method?

File: common/src/main/java/org/talend/components/common/text/SchemaUtils.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        return getUniqueName(result, previousNames);
+    }
+
+    private static String getUniqueName(String name, Set<String> previousNames) {
+        int index = 0;
+        String currentName = name;
+        while (previousNames.contains(currentName)) {
+            currentName = name + (++index);
+        }
+        return currentName;
```
Comment (undx) [inline common/src/main/java/org/talend/components/common/text/SchemaUtils.java:62]: already seen this code in the connectors-se code base. Maybe, later factorize to an uniq place...

File: common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelRecordReader.java (hunk 2 above)
Comment (clesaec) [reply common-stream-io/stream-excel/src/main/java/org/talend/components/common/stream/input/excel/ExcelRecordReader.java:53]: No, view with Yves, we keep abstract method in interface (no default) to oblige to think about a potentially close method.

File: common/src/main/java/org/talend/components/common/text/SchemaUtils.java (hunk 3 above)
Comment (clesaec) [reply common/src/main/java/org/talend/components/common/text/SchemaUtils.java:62]: Yes !! Here, it's move in "common" place :)

**Pull request merged**